src/components/file/file.tsx

```diff
@@ -119,17 +119,35 @@ export class File {
     private interact: EventEmitter<number | string>;
 
     public render() {
-        return (
+        return [
             <limel-file-dropzone
                 disabled={this.disabled || this.readonly || !!this.value}
                 accept={this.accept}
                 onFilesSelected={this.handleNewFiles}
             >
                 {this.renderChipset()}
-            </limel-file-dropzone>
+            </limel-file-dropzone>,
+            this.renderDragAndDropTip(),
+        ];
+    }
+
+    private renderDragAndDropTip() {
+        if (this.value || this.disabled || this.readonly) {
+            return;
+        }
+
+        return (
+            <span
+                class="drag-and-drop-tip"
+                dropzone-tip={this.dropZoneTip()}
+            ></span>
```

Comment:
```suggestion
           return <span class="drag-and-drop-tip">{this.dropZoneTip()}</span>;
```

Here we don't need to do it the hacky way we used to do in the previous versions, where the solution was inside `limel-chip-set`. I don't even remember why we did it that we back then. I guess it must have been because of the lack of my typescript knowledge, or because we didn't want to mess around too much in MDC components, and not make updating them harder.

But right now, there is no reason to do this using pseudo elements and `content: attr(dropzone-tip);` really. We can just write this as the content of the `<span>`. I mean, we added the span itself. So we can add its content too. 

This of course means updating the CSS respectively.

Reply:
That's how I started, - with just text in span 😀 , then checked how we did it before. 🙃
Sure! I will add a fixup.